### PR TITLE
Optimize: simplified regexp

### PR DESCRIPTION
**Pull request by xiaoliwang**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
simplified regexps

#### Which issue(s) this PR fixes:
Fixes #

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:
```docs
NONE
```

### CODE REVIEW

**Review by neolit123 — COMMENTED** [review]
are all of these regexp covered by unit tests?

**Comment by xiaoliwang** [comment]
> are all of these regexp covered by unit tests?

yes

File: cmd/kubeadm/app/preflight/utils.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ import (
 
 // GetKubeletVersion is helper function that returns version of kubelet available in $PATH
 func GetKubeletVersion(execer utilsexec.Interface) (*version.Version, error) {
-	kubeletVersionRegex := regexp.MustCompile(`^\s*Kubernetes v((0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)([-0-9a-zA-Z_\.+]*)?)\s*$`)
+	kubeletVersionRegex := regexp.MustCompile(`^\s*Kubernetes v((0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)([-.+\w]*)?)\s*$`)
```
Comment (SataQiu) [inline cmd/kubeadm/app/preflight/utils.go:31]:
```suggestion
	kubeletVersionRegex := regexp.MustCompile(`^\s*Kubernetes v((0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)([-\w.+]*)?)\s*$`)
```

**Comment by neolit123** [comment]
please squash the commits to one

**Comment by xiaoliwang** [comment]
> please squash the commits to one

OK

**Review by neolit123 — COMMENTED** [review]
/lgtm
/approve

looks correct, thanks

**Comment by neolit123** [comment]
actually tests are not running yet, so let's see

/ok-to-test

**Pull request merged**